packages/angular_devkit/core/src/utils/template.ts

```diff
@@ -143,6 +155,12 @@ export function templateParser(sourceText: string, fileName: string): TemplateAs
       children.push({ kind: 'escape', expression: escape, start, end } as TemplateAstEscape);
       start = end;
     }
+    if (comment) {
+      end = _positionFor(sourceText, offset + comment.length + 5);
```

Comment:
What's the meaning of `5` here? Just curious, I see other cases around this one also use it.

Reply:
Number of characters taken from the template itself (the `<%?` and `%>` parts).